data/blog/2025/2025-02-03-summary-of-the-meeting-in-february-2025.md

```diff
@@ -0,0 +1,59 @@
+---
+id: 2025-02-03-summary-of-the-meeting-in-february-2025
+author: bidi
+title: 'Summary of the meeting in February 2025'
+draft: false
+public: true
+created: '2025-02-04T11:00:00-01:00'
+updated: '2025-02-04T11:00:00-01:00'
+tags:
+    - meeting summary
+---
+
+On Monday, 3 February 2025, the Technical Steering Committee for the Laminas Project held its monthly meeting.
+The topics discussed included:
+
+- Creating a major release for laminas/laminas-session.
+- Setting up email accounts for @getlaminas.org.
+- Dropping Support for PHP 8.1 in future releases for Laminas packages.
+- New design and layout for getlaminas.com.
+
+<!--- EXTENDED -->
+# Creating a major release for [laminas/laminas-session](https://github.com/laminas/laminas-session)
+
+Backward compatibility among dependencies is desirable to maintain support for as many versions as is reasonable.
+Removing dependencies or limiting them to more recent versions has direct impact on all applications that use the older version.
+Releasing a backward incompatible version means some users may not be able to update to it without reworking the whole list of 3rd party packages.
+Eventually, releases that break compatibility are inevitable to benefit from new features, bugfixes and security updates.
+
+The TSC members agreed that a major release is needed for `laminas/laminas-session` because the update breaks backward compatibility.
+The `laminas/laminas-session` package currently prevents other packages from being updated, because of its dependency for [laminas/laminas-servicemanager](https://github.com/laminas/laminas-servicemanager) version 3.
+Updating the dependency to `laminas/laminas-servicemanager` version 4 would allow packages that depend on laminas-session to also benefit from the newer servicemanager.
+
+# Setting up email accounts for @getlaminas.org
+
+[Tuta](https://tuta.com/blog/tutanota-for-open-source-teams) offers a free plan for setting up secure emails for a custom domain.
+There are multiple benefits to using Tuta, but the Laminas TSC members didn't find any use care that would justify setting it up for getlaminas.org.
+An alternative is already in use within the Laminas organization for security and marketing concerns, so using Tuta was put on hold.
+
+# Dropping Support for PHP 8.1 in future releases for Laminas packages
+
+[Psalm](https://github.com/vimeo/psalm) and [PHPUnit](https://github.com/sebastianbergmann/phpunit) have released or are working on releases that remove support for PHP version 8.1.
```

Comment:
Psalm hasn't dropped support for 8.1 and there's nothing that states they plan to at the moment. Psalm v6 enables us to use more recent versions of PHPUnit, but recent versions of PHPUnit are 8.2+

Reply:
I reworked this section with more details on versions and PHP restrictions. ultimately, there is no reason to not update Psalm to v6. PHPUnit must stay at v10, though